examples_tests/06.MeshLoaders/mesh.frag

```diff
@@ -1,8 +1,8 @@
-#version 330 core
+#version 450 core
```

Comment:
dont you need a different directive for Vulkan GLSL ?

Reply:
hm?

Reply:
@Crisspl  r.f.c.?

Reply:
?

Reply:
what #extension? i dont know whats going on

Reply:
`KHR_vulkan_glsl` ?

Reply:
all right.